card/mod/01_core/spec/set/all/fetch_spec.rb

```diff
@@ -97,6 +97,23 @@
         a.save!
       end
     end
+    describe 'default option' do
+      context "when card doesn't exist" do
+        it 'initializes new cards' do
+          card = Card.fetch 'non-existent',
+                            new: { default_content: 'default content' }
+          expect(card.content).to eq 'default content'
+        end
+      end
+      context "when new card exist" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.